samples/nrf9160/at_monitor/src/main.c

```diff
@@ -44,13 +44,15 @@ static void cereg_mon(const char *notif)
 {
 	int ret;
 
-	ret = sscanf(notif, "+CEREG: %d", &cereg_status);
-	if (ret != 1) {
-		printk("Could not parse notification (%d, status: %d)\n",
-		       ret, cereg_status);
+	/* Make sure notification formats as we expect. */
+	ret = strncmp(notif, "+CEREG: ", 8);
```

Comment:
@lemrey Do you think it is necessary to do the string compare? The at monitor will trigger on CEREG, so it is only to check "+" and ": ".

Reply:
No, it shouldn't be necessary